plugins/modules/remote_management/manageiq/manageiq_policies.py

```diff
@@ -42,7 +42,11 @@
   resource_name:
     description:
       - the name of the resource to which the profile should be [un]assigned
-    required: true
+    required: only if resource_id is not set, mutually exclusive
+  resource_id:
+    description
```

Comment:
```suggestion
    description:
```
This causes the YAML to be invalid and most of the sanity check errors.